spec/integration/app_spec.rb

```diff
@@ -127,8 +127,17 @@ def start_app(cmd, port, timeout)
           File.exist?(mongoid_config_file).should be true
 
           config_text = File.read(mongoid_config_file)
-          config_text.should =~ /mongoid_test_config_development/
-          config_text.should =~ /mongoid_test_config_test/
+          expect(config_text).to match /mongoid_test_config_development/
+          expect(config_text).to match /mongoid_test_config_text/
```

Comment:
Would suggest to add a verbatim assertion here of the format of one full option that is not likely to change, e.g. 

```
text = <<~OPT
    # Mark belongs_to associations as required by default, so that saving a
    # model with a missing belongs_to association will trigger a validation
    # error. (default: true)
    # belongs_to_required_by_default: true
OPT
expect(config_text).to include(text)
```

Reply:
That seems very reasonable. I'll take a closer look.